docs/v3.x/plugins/users-permissions.md

```diff
@@ -160,13 +160,12 @@ For better understanding, you may find as follows the description of the login f
 
 #### Understanding the login flow
 
-Let's say that strapi's backend is located at: strapi.website.com.
-Let's say that your app frontend is located at: website.com.
+Let's say that strapi's backend is located at: strapi.website.com and your app frontend is located at: website.com.
```

Comment:
Can you please change this back to two lines as I believe it to be easier to read.